Visual Oddball Target.md

```diff
@@ -0,0 +1,35 @@
+
+# References
+Main Paper: https://psycnet.apa.org/fulltext/2015-45614-001.html
+
+Supplemental Materials: https://supp.apa.org/psycarticles/supplemental/xhp0000145/xhp0000145_supp.html
+
+# Stimuli
+## The visual components
+![imagesVisual_Oddball_Target_205](https://user-images.githubusercontent.com/89650778/132869370-e674fe0f-4937-4441-9187-6e065a478c20.jpg)
+
+# Procedure
+## Steps
+1. Both dyad members sat in the same room and performed a visual search for a contrast odd ball on stimuli presented on separate monitors. In each trial, two visual search arrays were presented in consecutive intervals separated by a blank screen. One of the two search arrays contained a contrast oddball item. Target contrast varied randomly between trials and was determined by addition of any one of 0.01, 0.03, 0.07, or 0.15 to the distractor baseline. The task was to decide in which interval the target appeared. Target interval was randomized across trials.
+2. In both experiments each trial was initiated by the participant responding with the keyboard. A black central fixation cross appeared on the screen for a variable period, drawn uniformly from the range 500–1000 ms. The two observation intervals were separated by a blank display lasting 1000ms. The fixation cross turned into a question mark after the second interval to prompt the participants to respond. 
+3. In Experiment 1, dyads of participants performed a visual oddball search task together. Both participants had identical displays in which stimuli and feedback were presented in the centre of the screen. In the Independent condition participants initially made a private decision. If they disagreed, discussion and collective decision ensued. In the Influence condition, no private decisions were made and collective decision was immediately negotiated.
+4. In Experiment 2, each participant viewed one half of their screen: the left half of one display for the participant responding with the keyboard, and the right half of the other display for the participant responding with the mouse. A piece of thick black cardboard was used to occlude the half not viewed by the participant. Two stimulus arrays were presented on both displays simultaneously, each on one-half of the display.
+
+## Roles 
+In Experiment 2, prior to the main test, each participant completed 48 trials of an isolated version of the same visual search task. In this measurement, target contrast was determined dynamically using a 2-up-1-down staircase procedure to determine the participant’s contrast sensitivity, which allowed to determine the more- and less-sensitive members of each dyad prior to starting the main experiment. White noise was added to the display of the less-sensitive participant, making it harder to detect the target. This procedure ensured that a substantial sensitivity gap was inserted between the two members of each dyad, leading to failure of collective decision making.
```

Comment:
I feel like this can be added under one of the procedure steps, or even to stimuli. Roles can be "Each participant was responsible for viewing the visual search display and making a decision.  In the IND condition, the decision was made independently, while in the INF condition, the decision was made collectively."

Reply:
Moved this as the fourth step and added roles as you suggested